src/words.ts

```diff
@@ -16,8 +16,12 @@ export const words = async (message: Message) => {
   if (message.author.bot) {
     return;
   }
-
-  const match = alex.markdown(message.content, ALEX as alex.Config).messages;
+  // Regex to remove puntuation from text
+  const textAfterRemovingPunctuation = message.content
```

Comment:
i feel like the regex matching should be a function.
Also should note how we are matching in a comment.

```ts
function stripSpecialCharacters(str) {
  // match characters . , and replace with ' '
  return str.replace(/[.,]/g, ' ').replace( // regex here );
}

There is more than one way to do this. What do you think?
I'd like to hear another opinion as well.

Reply:
Having it in a function would make this neater and easier to edit 👍

Reply:
There are multiple ways to do this, however, I found this regex method to be better than string processing since this would be slightly easier to modify in future and took in account most corner cases 🤔